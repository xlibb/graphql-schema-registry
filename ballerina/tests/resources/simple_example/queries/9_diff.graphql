query Diff {
    diff(newVersion: "0.1.1", oldVersion: "0.1.0") {
        severity
        action
        subject
        location
        value
        fromValue
        toValue
    }
}
