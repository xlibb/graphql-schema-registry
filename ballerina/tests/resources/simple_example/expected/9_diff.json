{
    "data": {
        "diff": [
            {
                "severity": "SAFE",
                "action": "ADDED",
                "subject": "TYPE",
                "location": [],
                "value": "User",
                "fromValue": null,
                "toValue": null
            },
            {
                "severity": "SAFE",
                "action": "ADDED",
                "subject": "FIELD",
                "location": [
                    "Query"
                ],
                "value": "users",
                "fromValue": null,
                "toValue": null
            }
        ]
    }
}